jsonexamples/small/jsoniter_scala/README.md

```diff
@@ -1,4 +1,28 @@
 Files from https://github.com/plokhotnyuk/jsoniter-scala/tree/master/jsoniter-scala-benchmark/src/main/resources/com/github/plokhotnyuk/jsoniter_scala/benchmark
 
-See issue "Lower performance on small files":
+See issue:
 https://github.com/lemire/simdjson/issues/70
+
+The files che-*.geo.json are number-parsing stress tests.
+
+```
+$ for i in *.json ; do echo $i; ./parsingcompetition $i  ; done
```

Comment:
I added the ability to pass multiple files to parsingcompetition, IIRC, so you should be able to do `./parsingcompetition *.json`. Small thing, and this documentation is fine, just figured you'd want to know.

Reply:
Noted